app/views/publishers/vacancies/summary.html.slim

```diff
@@ -1,26 +1,32 @@
-- content_for :page_title_prefix, t("jobs.confirmation_page.page_title", organisation: current_organisation.name)
+- content_for :page_title_prefix, t(".page_title", organisation: current_organisation.name)
 
-.vacancy.govuk-grid-row
+.govuk-grid-row
   .govuk-grid-column-two-thirds
-    .govuk-panel.govuk-panel--confirmation.submit-listing-success-gtm
+    = govuk_panel title: t(".success"), body: ""
```

Comment:
does `, body: ""` needs to be passed here?

Reply:
Yes